packages/core-data/src/resolvers.js

```diff
@@ -25,15 +25,37 @@ import {
 import { getKindEntities, DEFAULT_ENTITY_KEY } from './entities';
 import { apiFetch, select, resolveSelect } from './controls';
 import { ifNotResolved, getNormalizedCommaSeparable } from './utils';
+import getQueryParts from './queried-data/get-query-parts';
 
 /**
  * Requests authors from the REST API.
+ *
+ * @param {Object} query Query object.
+ */
+export function* getAuthors( query = {} ) {
+	const path = addQueryArgs(
+		'/wp/v2/users/?who=authors&per_page=100',
+		query
+	);
+
+	const users = yield apiFetch( {
+		path,
+	} );
+	const { stableKey } = getQueryParts( query );
+
+	yield receiveUserQuery( stableKey, users );
+}
+
+/**
+ * Request a single author from the REST API.
+ *
+ * @param {string} id The ID of the author to retrieve.
  */
-export function* getAuthors() {
+export function* getAuthor( id = '' ) {
```

Comment:
Is the default value necessary here?

Reply:
Actually, do we need this selector/resolver at all, why not just use `getUser` ?

Reply:
Not sure I looked at getUser, I'll try that.